README.md

```diff
@@ -1,13 +1,55 @@
 
+## Overview
+The adyen-integration connector contains two modules :  
+- Enabler: Acts as a wrapper implementation in which frontend components from Adyen embedded. It gives control to checkout product on when and how to load the connector frontend based on business configuration. In cases connector is used directly and not through Checkout product, the connector library can be loaded directly on frontend than the PSP one.
+- Processor : Acts as backend services which is middleware to integrate with Adyen platform. It is mainly responsible for managing transactions with Adyen and updating payment entity in composable commerce.  `connect-payment-sdk` will be offered to be used in connector to manage request context, sessions and other tools necessary to transact.
+
+```mermaid
+%%{ init : { "theme" : "", "flowchart" : { "defaultRenderer": "dagre-wrapper", "curve" : "linear" }}}%%
+
+flowchart TD
+    node1(( ))
+    node2(( ))
+    user("User")-->checkout("Commercetools Checkout")
+    subgraph connector
+        enabler
+        processor
+    end
+    subgraph coco["Commercetools Composable Commerce"]
+        cart
+        order
+        payment
+        
+    end
+    checkout----node1
+    
+    node1--"1. Cart management"-->cart("cart")
+    node1--"2. Order management"-->order("order")
+    checkout("Commercetools Checkout")----node2
+    node2--"3. Get static assets"-->enabler("enabler")
+    node2--"4. Request payment \noperations"-->processor("processor")
+    processor--"5. Request payment \noperations"-->psp("Adyen payment service provider")
+    psp("Adyen payment service provider")--"6. Return response"-->processor
+    processor--"7. Payment management"-->payment("payment")
+```
+
+1. commercetools Checkout in front-end side sends requests to composable commerce for cart creation.
+2. commercetools Checkout retrieves SDK as static assets from enabler in connector.
+3. After downloading the SDK, commercetools Checkout sends request via the SDK to endpoints exposed by processor for various payment operations.
+4. The processor transforms the received request and send them to Ayden platform.
+5. Adyen processes the payment operations and return the response to processor.
+6. The processor handles the payment persistence to composable commerce, and return the result to front-end.
+7. commercetools Checkout handles the order management in composable commerce once it receives the result from the processor. 
+
 ## Prerequisite
 #### 1. commercetools composable commerce API client
 Users are expected to create API client responsible for payment management in composable commerce project. Details of the API client are taken as input as environment variables/ configuration for connect such as `CTP_PROJECT_KEY` , `CTP_CLIENT_ID`, `CTP_CLIENT_SECRET`, `CTP_REGION`. For details, please read [Deployment Configuration](./README.md#deployment-configuration).
```

Comment:
Can we remove `CTP_REGION` from here as well?

Reply:
https://github.com/commercetools/connect-payment-integration-adyen/pull/76/commits/491229dfad08babb346fe79a5bad76c9d08f9a73